sysrib/server.go

```diff
@@ -239,19 +239,19 @@ func (s *Server) monitorConnectedIntfs(yclient *ygnmi.Client) error {
 				if intf.Enabled != nil {
 					if intf.Ifindex != nil {
 						ifindex := intf.GetIfindex()
-						s.setInterface(name, int32(ifindex), intf.GetEnabled())
+						s.setInterface(context.Background(), name, int32(ifindex), intf.GetEnabled())
```

Comment:
is there a context that can be passed here?

have everything take a context will make reset easier to implement